docs/content/concepts/assistant.md

```diff
@@ -0,0 +1,186 @@
+---
```

Comment:
Grabbing this from other PR so I can display Assistant in Nav

Reply:
If we can merge this change ealier than the assistant page PR, removing this from this PR, merging this PR without assistant page, and then adjusting the assistant page PR to be compatible with this new structure may be smoother